cypress/e2e/navigation.cy.ts

```diff
@@ -35,11 +66,263 @@ describe('Navigation', () => {
+    });
+
+    it('should navigate by clicking folder items in content view', () => {
+        const publicDriveName = Cypress.env('TEST_PUBLIC_DRIVE_NAME') as string;
+        selectSidebarItem(publicDriveName);
+
+        newFolder(publicDriveName, 'test-folder');
+        newFolder('test-folder', 'test-folder-2');
+        newFolder('test-folder-2', 'test-folder-3');
+        newFolder('test-folder-3', 'test-folder-4');
+
+        selectSidebarItem('test-folder');
+
+        clickContentViewItem('test-folder-2');
+        clickContentViewItem('test-folder-3');
+        clickContentViewItem('test-folder-4');
+
+        clickSidebarItemOption('test-folder', 'delete');
+        cy.contains('button', 'Delete').click();
+    });
+
+    it('should navigate using the breadcrumb', () => {
+        const publicDriveName = Cypress.env('TEST_PUBLIC_DRIVE_NAME') as string;
+        selectSidebarItem(publicDriveName);
+
+        newFolder(publicDriveName, 'test-folder');
+        newFolder('test-folder', 'test-folder-2');
+        newFolder('test-folder-2', 'test-folder-3');
+        newFolder('test-folder-3', 'test-folder-4');
+
+        selectSidebarItem('test-folder-4');
+
+        getBreadcrumbItem('test-folder-3').should('be.visible').click();
+        getBreadcrumbItem('test-folder-4').should('not.exist');
+
+        getBreadcrumbItem('test-folder-2').should('be.visible').click();
+        getBreadcrumbItem('test-folder-3').should('not.exist');
+
+        getBreadcrumbItem('test-folder').should('be.visible').click();
+        getBreadcrumbItem('test-folder-2').should('not.exist');
+
+        getBreadcrumbItem(publicDriveName).should('be.visible').click();
+        getBreadcrumbItem('test-folder').should('not.exist');
+
+        clickSidebarItemOption('test-folder', 'delete');
+        cy.contains('button', 'Delete').click();
+    });
+
+    it('should open switchboard from document link', () => {
+        const rwaDocumentCloseSelector =
+            '#document-editor-context > div > div.flex.items-center.justify-between > div.flex.justify-end.gap-x-2 > button.grid.size-8.place-items-center.rounded.border.border-gray-200.active\\:opacity-50';
+        const rwaSwitchboardLinkSelector =
+            '#document-editor-context > div > div.flex.items-center.justify-between > div.flex.justify-end.gap-x-2 > button:nth-child(1)';
+        const publicDriveName = Cypress.env('TEST_PUBLIC_DRIVE_NAME') as string;
+        selectSidebarItem(publicDriveName);
+
+        newFolder(publicDriveName, 'test-folder');
+        selectSidebarItem('test-folder');
+
+        cy.contains('RWA Portfolio').click();
+        cy.get('input[placeholder="Document name"]')
+            .clear()
+            .type('rwa-document');
+        cy.get('button').contains('Create').click();
+
+        cy.window().then(win => {
+            cy.stub(win, 'open').as('windowOpen');
+        });
+
+        cy.get(rwaSwitchboardLinkSelector).click();
```

Comment:
Is it possible to check if the query on switchboard returns the state?

Reply:
absolutely, I don't know why the link isn't working in develop env, but I'll check with Frank and add a new test for this 👍🏻